tests/console-backend-service/internal/domain/k8s/secret_test.go

```diff
@@ -267,31 +280,31 @@ func fixDeleteSecretMutation() *graphql.Request {
 	return req
 }
 
-func secretEvent(eventType string, secret secret) SecretEvent {
-	return SecretEvent{
-		Type:   eventType,
-		Secret: secret,
-	}
-}
-
-func readSecretEvent(sub *graphql.Subscription) (SecretEvent, error) {
-	type Response struct {
-		SecretEvent SecretEvent
-	}
-	var secretEvent Response
-	err := sub.Next(&secretEvent, tester.DefaultSubscriptionTimeout)
-
-	return secretEvent.SecretEvent, err
-}
-
-func checkSecretEvent(expected SecretEvent, sub *graphql.Subscription) error {
-	for {
-		event, err := readSecretEvent(sub)
-		if err != nil {
-			return err
-		}
-		if expected.Type == event.Type && expected.Secret.Name == event.Secret.Name {
-			return nil
-		}
-	}
-}
+//func secretEvent(eventType string, secret secret) SecretEvent {
```

Comment:
Why is it commented?

Reply:
We are disabling secret subscriptions temporarily due to security reasons.